modules/e2ee/OlmAdapter.js

```diff
@@ -533,6 +829,12 @@ export class OlmAdapter extends Listenable {
                     this._sendMessage(data, participantId);
                 }
             }
+            break;
+        case 'e2ee.idKey.ed25519':
+            olmData.sasVerification = olmData.sasVerification ?? {};
+
+            olmData.sasVerification.ed25519 = newValue;
```

Comment:
Here we should probably emit some "verification possible / available / supported" event.